packages/mask/dashboard/Dashboard.tsx

```diff
@@ -70,9 +70,10 @@ export default function Dashboard() {
         <>
             <CssBaseline />
             {GlobalCss}
-            {process.env.NODE_ENV === 'development' ?
+            {/* https://github.com/TanStack/query/issues/5417 */}
+            {/* {process.env.NODE_ENV === 'development' ?
                 <ReactQueryDevtools buttonPosition="bottom-right" />
-            :   null}
+            :   null} */}
```

Comment:
Why comment out `ReactQueryDevtools`? @Jack-Works

Reply:
it is still crashing see https://github.com/TanStack/query/issues/5417